search-api/openshift/templates/dc.json

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
This template is the same as search-api-deploy.json. You may not need this one for now. In auth, I created this template for new CD process.